CONTRIBUTING.md

```diff
@@ -0,0 +1,529 @@
+section within each tutorial should be included in a markdown cell at the very
+end of the notebook.  This cell should contain a citation list of the form
+above.  Since the tutorials are effectively standalone entities, however, this
+`Tutorial References` section should *not* be augmented with a central
+references section within the method's README.  Additionally, specific
+equations should be referenced within tutorials using the citation key for the
+appropriate publication.
+
+### Header Styleguide
+
+#### Reference Implementation Headers
+
+For reference implementations, each file should begin with a statement about the purpose
+of the script, author and contributor information, and citations for algorithms
+and/or publications from which the implementation was created:
+
+```
+#* A simple explanation of script contents
+#*
+#* Author: <auth1>
+#* Additional Contributors: <auth2>, <auth3>
+#* Date: <most-recent-revision>
+#*
+#* Algorithms & equations taken from <ref1> and <ref2>.
+```
+
+The difference between `Author` and `Additional Contributor` is a fine line;
+one possible delineation is that `Author` may refer to an individual who wrote
+large portions of the script, while `Additional Contributor` may refer to an
+individual who cleaned up or modified the code slightly.
+
+Citation keys <ref1> and <ref2> should take the form `[LNFA:yy:pp]`, as
+detailed [above](#citation-styleguide). This `[LNFA:yy:pp]` key should match
+with the entry in the main `README.md` file for the method.   
+
+#### Tutorial Headers
+
+The first cell of an interactive tutorial should be a Markdown (md) cell, with the
+following contents (given below in raw Markdown):
+
+```
+# Title of Tutorial
+
+Original Author: <auth1>
+Additional Contributors: <auth2>, <auth3>
+Date: <most-recent-revision>                             
+
+Short paragraph detailing the contents, goals, and/or organization of the tutorial.
+```
+
+Much more than reference implementations, interactive tutorials are likely to
+evolve over time as it is added to or modified by various other contributors.
+Therefore, the `Primary Author` field from the reference implementation header
+has been replaced with `Original Author`, so (i) that this semial contribution
+is not disregarded, and (ii) the additional contributions by others are not
+devalued.
+
+### Documentation Styleguide
+
+Documentation for the contents of the Psi4NumPy repository is contained in
+`README.md` files in each directory.  
+
+For reference implementations, `README.md` files in the `method-dir` should contain
+* a *very* brief overview of the most important aspects of the theory,
+* a description of the proper use and capabilities of any helper classes or modules, and
+* a list containing relevant citations for all scripts in the `method-dir` (see [above](#citation-styleguide))
+See the SAPT README [here](https://github.com/psi4/psi4numpy/blob/master/Symmetry-Adapted-Perturbation-Theory/README.md)
+for an example.
```

Comment:
Isn't the main niftiness of the SAPT README the definition of the primary arrays/variables?